airbyte-cdk/python/local-cdk-requirements.txt

```diff
@@ -0,0 +1,2 @@
+pandas==2.0.3
```

Comment:
Curious, even if work in progress, why not just add them to the pyproject.toml temporarily?